src/main/controllers/features/limited/companyInactiveController.ts

```diff
@@ -0,0 +1,18 @@
+import { NextFunction, Request, Response } from "express";
+import * as config from "../../../config";
+import { selectLang, addLangToUrl, getLocalesService, getLocaleInfo } from "../../../utils/localise";
+import { LIMITED_WHAT_IS_THE_COMPANY_NUMBER, LIMITED_COMPANY_INACTIVE, START, BASE_URL } from "../../../types/pageURL";
+
+export const get = async (req: Request, res: Response, next: NextFunction) => {
+    const lang = selectLang(req.query.lang);
+    const locales = getLocalesService();
+    const companyName = "Comany Name";
```

Comment:
I believe a service class should be created which has a method like getCompanyName().
This method should get the value from session. For now you can make it return default value.